### ISSUE TITLE
BUG: shorcuts/hotkeys dont work in guest-iframe

### ISSUE THREAD

**Issue by mhsdesign** [neos/neos-ui#3099]
### Description

shorcuts/hotkeys dont work when focus is set to the guest-iframe (last mouse click was there)

### Steps to Reproduce

1. open the NeosUI Demo
2. klick into the iframe (not an editable element, but just somewhere 'safe' into the body)
3. type any shorcut like `t f`

#### Expected behavior

shortcut is executed

#### Actual behavior

shortcut does only work, if one previously clicked into the main frame someplace

### Code
looking quickly over `handleHotkeys`, this behaviour should be allowed (unless one is inline editing) ...
https://github.com/neos/neos-ui/blob/6ce277470258599c1d29f9ec2f5983e7e44d4338/packages/neos-ui-sagas/src/UI/Hotkeys/index.js#L6

### Affected Versions

UI: 8.0 but also 7.3 and maybe other versions

Chromium Edge 100.0.1185.36  & Firefox  97.0.1
